{
	"schema_version": "1.6.0",
	"id": "UBUNTU-CVE-2018-11410",
	"modified": "2018-05-24T00:00:00Z",
	"published": "2018-05-24T00:00:00Z",
	"related": [
		"CVE-2018-11410",
		"USN-3669-1"
	],
	"details": "An issue was discovered in Liblouis 3.5.0. A invalid free in the compileRule function in compileTranslationTable.c allows remote attackers to cause a denial of service (application crash) or possibly have unspecified other impact.",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Ubuntu:18.04:LTS",
				"name": "liblouis",
				"purl": "pkg:deb/ubuntu/liblouis?arch=src?distro=bionic"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.5.0-1ubuntu0.1"
						}
					]
				}
			],
			"versions": [
				"3.0.0-3ubuntu1",
				"3.3.0-1",
				"3.4.1-1",
				"3.5.0-1"
			],
			"ecosystem_specific": {
				"availability": "No subscription required",
				"binaries": [
					{
						"binary_name": "liblouis-bin",
						"binary_version": "3.5.0-1ubuntu0.1"
					},
					{
						"binary_name": "liblouis-bin-dbgsym",
						"binary_version": "3.5.0-1ubuntu0.1"
					},
					{
						"binary_name": "liblouis-data",
						"binary_version": "3.5.0-1ubuntu0.1"
					},
					{
						"binary_name": "liblouis-dev",
						"binary_version": "3.5.0-1ubuntu0.1"
					},
					{
						"binary_name": "liblouis14",
						"binary_version": "3.5.0-1ubuntu0.1"
					},
					{
						"binary_name": "liblouis14-dbgsym",
						"binary_version": "3.5.0-1ubuntu0.1"
					},
					{
						"binary_name": "python-louis",
						"binary_version": "3.5.0-1ubuntu0.1"
					},
					{
						"binary_name": "python3-louis",
						"binary_version": "3.5.0-1ubuntu0.1"
					}
				],
				"ubuntu_priority": "low"
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/cve/2018/UBUNTU-CVE-2018-11410.json"
			}
		}
	],
	"references": [
		{
			"type": "REPORT",
			"url": "https://ubuntu.com/security/CVE-2018-11410"
		},
		{
			"type": "REPORT",
			"url": "https://docs.google.com/document/d/1Uw3D6ECXZr8S2cWOTY81kg6ivv0WpR4kQqxVpUSyGUA/edit?usp=sharing"
		},
		{
			"type": "REPORT",
			"url": "https://ubuntu.com/security/notices/USN-3669-1"
		},
		{
			"type": "REPORT",
			"url": "https://www.cve.org/CVERecord?id=CVE-2018-11410"
		}
	]
}
